api/controllers/SignupController.js

```diff
@@ -72,12 +72,29 @@ module.exports = {
 
         // Result from an update is an array
         if (result.length > 0) {
-          return res.json(result[0]);
+          return result[0];
         } else {
           // Result from a create is the user object
-          return res.json(result);
+          return result;
         }
       })
+      .then(function(result) {
+        // Create custom referral url after a sms user has been created
+        // Use a users first name, phone number or email
+        let uniqueString = result.firstName || result.email || result.phone;
```

Comment:
Of a user's info, we'll only want to use the first name here. If it doesn't exist, I think it'd be safer to default to something like `SHINE` or some other lexicon strategy as opposed to exposing email and phone.